homeassistant/components/camera/local_file.py

```diff
@@ -29,11 +28,6 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Camera."""
     file_path = config[CONF_FILE_PATH]
 
-    # check filepath given is readable
-    if not os.access(file_path, os.R_OK):
```

Comment:
Can you leave this check here but instead emit a warning that the file was not readable.